subprojects/core/src/main/java/org/gradle/invocation/DefaultGradle.java

```diff
@@ -109,22 +108,6 @@ public DefaultGradle(@Nullable BuildState parent, StartParameter startParameter,
         buildListenerBroadcast = getListenerManager().createAnonymousBroadcaster(BuildListener.class);
         projectEvaluationListenerBroadcast = getListenerManager().createAnonymousBroadcaster(ProjectEvaluationListener.class);
 
-        buildListenerBroadcast.add(new InternalBuildAdapter() {
-            @Override
-            public void projectsLoaded(Gradle gradle) {
```

Comment:
❓ Previously, we executed this action before any callbacks registered by users via `org.gradle.BuildListener.projectsLoaded`. Is this still the case? Do we have a test that demostrates that directly or indirectly?

I guess, it's also a question of what happens if users apply plugins by id from the `projectsLoaded` callback

Reply:
Should we introduce a test to demonstrate the behavior is preserved?

Reply:
Introduced a test